pkg/storagev2/leveldbstore/store_test.go

```diff
@@ -18,3 +20,53 @@ func TestStoreTestSuite(t *testing.T) {
 	}
 	storagetest.TestStore(t, store)
 }
+
+func BenchmarkLevelDB(b *testing.B) {
+	st, err := ldb.New("", new(opt.Options))
+	if err != nil {
+		b.Fatal(err)
+	}
+	b.Cleanup(func() { _ = st.Close() })
+	storagetest.RunBenchmarkTests(b, st)
+}
+
+/* Benchmark results
```

Comment:
I think this result should be part of the PR description. Also, the results are relative, so it makes sense to compare them on the same computer.

Reply:
let's start a conversation on this topic, there's Alok's first comment to this PR is exactly about that, please reply  there and we'll find a compromise.